draft-ietf-tls-subcerts.md

```diff
@@ -206,9 +207,9 @@ draft-02
 
 # Solution Overview
 
-A delegated credential is a digitally signed data structure with two semantic
+A delegated credential (DC) is a digitally signed data structure with two semantic
```

Comment:
Is the acryonym DC used anywhere except the diagram?

Reply:
Yes! Sometimes, for example, on `DC-enabled`, `non-DC traffic`.